src/language/ast.js

```diff
@@ -44,6 +44,30 @@ export type Location = {
 };
 
 /**
+ * Represents the different kinds of tokens in a GraphQL document.
+ */
+export type TokenKind = '<SOF>'
```

Comment:
https://github.com/graphql/graphql-js/blob/da09c4ef1e5dd5ae2d99a73296a3274a03b609c7/src/language/lexer.js#L84-L131 already defines this - maybe we should just reuse?